LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Resources/JSON/fakebroker.com.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "DDG Fake Broker",
+  "url": "fakebroker.com",
```

Comment:
Just a note: Broker JSON now has an `optOutUrl` property. Don’t think this impacts anything, more so an FYI.